### PR TITLE
adjust button color and fix bad merge

### PR DESCRIPTION
**Pull request by piyalbasu**
Changing button colors to adhere to the pattern established in the Figma of using:

* button variant "secondary" for the main CTA
* button variant "tertiary" for the cancel/secondary CTA (for ex: "Got it" and "Cancel")
* button variant "primary" for actions that will cause changes to your account 

Some examples:

Screenshot 2024-11-05 at 12 29 53 PM
Screenshot 2024-11-05 at 12 36 40 PM
Screenshot 2024-11-05 at 12 36 11 PM
Screenshot 2024-11-05 at 12 35 49 PM

### CODE REVIEW

File: extension/src/popup/views/Account/index.tsx [hunk 1]
```diff
@@ -183,25 +180,7 @@ export const Account = () => {
             setLoading={setLoading}
           />
           {/* <BlockaidAnnouncement /> */}
-          <View.Content
```
Comment (piyalbasu) [inline extension/src/popup/views/Account/index.tsx:186]: this looks like it was a result of a bad merge. This button now appears in the `...` dropdown on Account

File: extension/src/popup/views/Account/styles.scss [hunk 2]
```diff
@@ -44,7 +44,7 @@
         stroke: var(--sds-clr-base-01);
         width: 1rem;
         height: 1rem;
-        stroke: var(--sds-clr-gray-11);
+        stroke: var(--sds-clr-gray-12);
```
Comment (piyalbasu) [inline extension/src/popup/views/Account/styles.scss:47]: fixing the button color of the `Send` and `...` buttons on Account

File: extension/src/popup/views/Account/index.tsx (hunk 1 above)
Comment (aristidesstaffieri) [reply extension/src/popup/views/Account/index.tsx:186]: ah yup that was my mistake, thanks!

File: extension/src/popup/views/Account/styles.scss (hunk 2 above)
Comment (aristidesstaffieri) [reply extension/src/popup/views/Account/styles.scss:47]: Screenshot 2024-11-05 at 10 45 41 AM

Hmm gray-11 loooks like this for me 🤔

**Review by aristidesstaffieri — APPROVED** [review]

**Pull request merged**